node-src/git/git.ts

```diff
@@ -281,3 +281,10 @@ export async function findFiles(...patterns: string[]) {
   const files = await execGitCommand(`git ls-files -z ${patterns.map((p) => `'${p}'`).join(' ')}`);
   return files.split('\0').filter(Boolean);
 }
+
+export async function mergeQueueBranchMatch(commit, branch) {
+  const mergeQueuePattern = new RegExp(`/pr-(\\d+)-${commit}$`);
```

Comment:
@tmeasday is this what you had in mind in terms of merge queue detection or is this too naive?

Reply:
More or less. Do you think we should check for the `gh-readonly-queue` part too?

Reply:
LOL for some reason I thought this was @ghengeveld's branch because of the gh and I wasn't sure how to get that from the envCI(). I see now it's a github prefix, updating now.